x-pack/plugin/build.gradle

```diff
@@ -138,6 +138,7 @@ tasks.named("yamlRestCompatTest").configure {
     'security/authz/14_cat_indices/Test explicit request while multiple authorized indices',
     'security/authz/14_cat_indices/Test explicit request while multiple opened/closed authorized indices',
     'security/authz/14_cat_indices/Test wildcard request with multiple authorized indices',
+    'security/authz/50_data_streams/Test that requests not supporting data streams do not include data streams among authorized indices',
```

Comment:
In 7.x, this test uses get alias api, but that api now supports data streams, 
so this yaml rest combat test needs to be muted.

Reply:
Hi @martijnvg I am testing in 7.14.0 with wildcard matching on the data streams indices and it does not seem to work. Not sure what I did wrong here
![image](https://user-images.githubusercontent.com/70958189/125849907-c3945613-44aa-469b-a50d-299a8ab3dd07.png)

![image](https://user-images.githubusercontent.com/70958189/125849978-999713ac-ca12-4a62-b7c9-6be77f136e60.png)

But not using wild card does work

![image](https://user-images.githubusercontent.com/70958189/125850098-a6347393-8eb3-4efb-ac50-7bf9a2837c29.png)

Reply:
Hey @hungnguyen-elastic, thanks for reporting! I checked this locally and ran into the same issue. I will work on a fix for this.

Reply:
I've opened #75456 to track this bug.

Reply:
Thank you @martijnvg

Reply:
@mikeh-elastic FYI